loginpage.htm

```diff
@@ -1,28 +1,36 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-	<title>Login</title>
-	<link rel="stylesheet" type="text/css" href="style.css">
-	<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" type="text/css" rel="stylesheet">
+    <title>Login</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" type="text/css" rel="stylesheet">
 </head>
 
 <body>
-<div class="Login">
-	<div class="heading">
-		<h1>Login</h1>
-	</div>
-	<form>
-  		<div class="form-group">
-   		 <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Email">
- 	    </div>
-  		<div class="form-group">
-   		 
-   		 <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
- 	    </div>
- 	    <hr>
-  		<button type="submit"  class="btn">Login</button>
-	</form>
-</div>
+    <div class="topnav">
```

Comment:
add home and signup
into login page nav bar

Reply:
added